client/src/components/card/Category.jsx

```diff
@@ -1,28 +1,37 @@
 import React, { useEffect, useState } from "react";
 import axios from "axios";
+import { useParams } from "react-router";
 
 const Category = () => {
-  const [apiList, setApiList] = useState([]);
+  const [categoryList, setCategoryList] = useState({});
+  const headers = Object.values(categoryList);
+  const { id, header } = useParams();
 
-  // useEffect(() => {
-  //   axios
-  //     // .get(`https://swapi.dev/api/${category}/${id}`)
-  //     .get(`https://bobsburgers-api.herokuapp.com/characters/`)
-  //     .then((response) => {
-  //       console.log("response =>", response);
-  //       setApiList(response.data);
-  //       // console.log(response.data);
-  //     })
-  //     .catch((err) => {
-  //       console.log("error!! ====> ", err);
-  //     });
-  // }, [setApiList]);
+  useEffect(() => {
+    axios
+      // characters will be dynamic
+      .get(`https://bobsburgers-api.herokuapp.com/${header}/`)
+      .then((response) => {
+        setCategoryList(response.data);
+      })
+      .catch((err) => {
+        console.log("error!! ====> ", err);
```

Comment:
Not urgent right now but you should start handling error cases in the UI instead of console logging.

Reply:
@jaygraygray like alert()??